index.js

```diff
@@ -74,17 +74,17 @@ function getKinesisData(event, callback) {
 function filterGDEvents(cwEvents, callback) {
     async.filter(cwEvents,
         function(cwEvent, filterCallback){
-            if ((cwEvent.source && 
+            var isValid = (cwEvent.source && 
```

Comment:
We should remove the first check `cwEvent.source &&`, otherwise `isValid` would be `undefined` if `source` field is absent in `cwEvent`.

Reply:
I don't think so.  It will be "false", won't it?